quicktests/template.html

```diff
@@ -26,4 +25,4 @@
     <div id="testdiv"></div>
   </body>
 
-</html>
+</html>
```

Comment:
Do a 
`git checkout develop -- quicktests/template.html`

since there's a diff here that's not that great (loss of EOF newline etc)